src/patternfly/components/Nav/nav.scss

```diff
@@ -186,6 +186,16 @@
   &::-webkit-scrollbar {
     display: none;
   }
+
+  // Disable underline
```

Comment:
is this meant to be part of this PR? I'd suggest making note of it if it is just to avoid confusion because it's part of a different component.

Reply:
Yes, I meant to roll that change into this PR. Talked to @andresgalante and moved these selectors under `.nav` to prevent a global text-decoration.

Reply:
👍